security/network_bound_disk_encryption/nbde-about-disk-encryption-technology.adoc

```diff
@@ -0,0 +1,30 @@
+// CNF-2127 assembly
+[id="nbde-about-disk-encryption-technology"]
+= About disk encryption technology
+include::modules/common-attributes.adoc[]
+:context: nbde-implementation
+
+toc::[]
+
+Network-Bound Disk Encryption (NBDE) allows you to encrypt root volumes of hard drives on physical and virtual
+machines without having to manually enter a password when restarting machines.
+
+include::modules/nbde-disk-encryption-technology-comparison.adoc[leveloffset=+1]
+
+include::modules/nbde-key-escrow.adoc[leveloffset=+2]
+
+include::modules/nbde-tpm-encryption.adoc[leveloffset=+2]
+
+include::modules/nbde-network-bound-disk-encryption.adoc[leveloffset=+2]
+
+include::modules/nbde-secret-sharing-encryption.adoc[leveloffset=+2]
+
+include::modules/nbde-using-tang-servers-for-disk-encryption.adoc[leveloffset=+1]
+
+include::modules/nbde-deciding-the-number-of-tang-servers-to-use.adoc[leveloffset=+2]
```

Comment:
This "sizing" topic does not need to be leveloffset+=2.  The [compute requirements](https://deploy-preview-36403--osdocs.netlify.app/openshift-enterprise/latest/security/network_bound_disk_encryption/nbde-tang-server-installation-considerations.html#nbde-compute-requirements_nbde-implementation) topic--based on title--seems like these two topics should be near each other.

Reply:
Changed to leveloffset+=1